V5/SpinUp/src/main.cpp

```diff
@@ -21,10 +21,101 @@ void pre_auton( void ) {
 }
 
 void autonomous( void ) {
-
+  
 }
 
 void usercontrol( void ) {
+  double driveMode = 0;
+  double driveSpeed = 0;
+  double flspeed = 0;
+  double frspeed = 0;
+  double blspeed = 0;
+  double brspeed = 0;
+  bool flfwd = true;
```

Comment:
Why do you need declare the boolean variables flfwd/frfwd/blfwd/brfwd and assign values here within usercontrol()? @coco131313